faucet/server/Main.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+{-# LANGUAGE RankNTypes        #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TypeOperators     #-}
+{-# OPTIONS_GHC -Wall #-}
+
+module Main where
+
+import           Control.Lens
+import           Control.Monad.Except
+import           Data.Aeson (eitherDecode)
+import           Data.ByteString.Lazy as BSL
+import           Network.Wai.Handler.Warp (run)
+import           Pos.Util.CompileInfo (withCompileInfo)
+import           Servant
+import           System.Environment (getArgs)
+import           System.Remote.Monitoring (forkServer, serverMetricStore)
+import           System.Remote.Monitoring.Statsd (forkStatsd)
+import           System.Wlog (LoggerNameBox, launchFromFile, usingLoggerName)
+
+import           Cardano.Faucet
+import           Cardano.Faucet.Swagger
+
+main :: IO ()
+main = withCompileInfo $ do
+    ekg <- forkServer "localhost" 8001
```

Comment:
This needs to be configurable with `--ekg-server` option, like the wallet.
You might be able to re-use option handling code from `Pos.Infra.Statistics.Ekg`.